jax/_src/compilation_cache_interface.py

```diff
@@ -14,12 +14,10 @@
 
 from abc import abstractmethod
 
-from jax._src import path as pathlib
 from jax._src import util
 
 
 class CacheInterface(util.StrictABC):
-  _path: pathlib.Path
```

Comment:
Good catch!

Reply:
Realised that this actually cannot be deleted because it is used in some tests